memory/src/main/java/org/opensearch/ml/memory/index/ConversationMetaIndex.java

```diff
@@ -366,7 +366,12 @@ public void updateConversation(UpdateRequest updateRequest, ActionListener<Updat
         }
         try (ThreadContext.StoredContext threadContext = client.threadPool().getThreadContext().stashContext()) {
             ActionListener<UpdateResponse> internalListener = ActionListener.runBefore(listener, () -> threadContext.restore());
-            client.update(updateRequest, internalListener);
+            client.admin().indices().refresh(Requests.refreshRequest(META_INDEX_NAME), ActionListener.wrap(refreshResponse -> {
```

Comment:
Why refresh first ? 

Suggest add refresh policy for `updateRequest` 
```
updateRequest.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE)
```

Reply:
Removed this one but the refresh is almost everywhere in the Memory layer by Aryn. We should clean them up if they affect the latency.